prog/prio_test.go

```diff
@@ -59,3 +59,50 @@ func TestPrioChoice(t *testing.T) {
 		t.Fatal(diff)
 	}
 }
+
+// Test static priorities assigned based on argument direction.
+func TestStaticPriorities(t *testing.T) {
+	target, rs, _ := initTest(t)
+	iters := 100000
```

Comment:
Use the iters returned by initTest. It accounts for short/non-short mode and race mode.
If you don't do it, the test either runs for loo long in short mode, or for too short in non-short mode. And probably too slow in race mode.
If you need to scale the iters you can always do:
```
iters *= 10 // our iters are very fast
```
or:
```
iters /= 5 // our iters are very slow
```